crypto/keys/src/ed25519.rs

```diff
@@ -201,6 +215,20 @@ impl TryFrom<&[u8]> for Ed25519Public {
     }
 }
 
+impl ReprBytes32 for Ed25519Public {
```

Comment:
`PUBLIC_KEY_LENGTH` is 32 bytes, so this works. If that was to change, this would stop compiling.